app/models/portal/offering_embeddable_metadata.rb

```diff
@@ -0,0 +1,10 @@
+class Portal::OfferingEmbeddableMetadata < ActiveRecord::Base
```

Comment:
I don't know if it is feasible with the time remaining on this work, but...

This model duplicates the associations of this model: `Report::EmbeddableFilter`.  I think it would make sense to add a 'filtered' property or perhaps a 'show_in_report' property to `OfferingEmbeddableMetadata`  and then perhaps Report::EmbeddableFilter could go away.

I remember that we have had problems with Report::EmbeddableFilter. If a question was removed from activity but the filter still referred to it, then it would crash the report. If a destroy relationship could be set up between the Embeddable and its OfferingMetadata, then this problem would go away.